src/oprf.ts

```diff
@@ -193,7 +202,8 @@ export class Evaluation {
         return res
     }
 
-    static deserialize(params: DLEQParams, bytes: Uint8Array): Evaluation {
+    static deserialize(suite: SuiteID, bytes: Uint8Array): Evaluation {
+        const params = Oprf.getDLEQParams(suite)
```

Comment:
```suggestion
        const params = suite.getDLEQParams()
```

Reply:
`suite` is light-weight string identifier, while Oprf is the object that charges with all the logic.